featurebyte/routes/target_table/controller.py

```diff
@@ -76,11 +76,19 @@ async def get_validation_parameters(
         feature_store = await self.feature_store_service.get_document(
             document_id=table_create.feature_store_id
         )
-        graph = table_create.graph
+        if table_create.target_id is not None:
+            target = await self.target_service.get_document(table_create.target_id)
+            graph = target.graph
+            nodes = [graph.get_node_by_name(target.node.name)]
+        else:
+            assert table_create.graph is not None
```

Comment:
Should we assert nodes is not None as well?

Reply:
`nodes` is a property that returns an empty list if the field `node_names` is None so it will never be None